### PR TITLE
MongDB Setup and Simple Test

### LINKED ISSUE #3: [Task] Connect MongoDB to the server

### PR DESCRIPTION
**Pull request by dlalaj**
# Description

PR to set up mongodb connection working with mongo server running locally. A simple test has been added to verify if the connection works succesfully. Instructions on how to run the test will be provided below.

References Issue #3 

# How Has This Been Tested?
Test were setup under `backend/simple_db_test` and can be run through the commands below. First follow the steps on the next section to ensure you have installed MongoDB and it has been started as a service
```commandline
cd simple_db_test
node test_model.js
```
If successful you should see printed on the terminal:
```commandline
TESTING CONNECTION!!
TESTING INSERT
Connection successful! 
{
  first_name: 'Denis',
  last_name: 'Lalaj',
  age: 23,
  ...
}
{
  first_name: 'Matt',
  last_name: 'Anderson',
  age: 36,
  ...
}
```
If you have [MongoCompass](https://www.mongodb.com/products/tools/compass) installed you can verify from the GUI that two documents were created with the data as shown above.

**Test Configuration**:
1. New dependencies were added so you should first run `npm install` for the new dependencies
2. You should ensure you have MongoDB installed as well. Follow the installation instructions for your OS [here](https://www.mongodb.com/docs/manual/administration/install-community/).
3. The MongoDB server should be up and running locally to serve requests coming our backend
```commandline
MacOS: 
###### brew services start mongodb-community@version
Ubuntu: 
###### systemctl start mongod.service 
###### systemctl enable mongod.service
Windows:
Starting MongoDB as a service is done during installation

```

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGODB_TEST_URI='mongodb://localhost:27017/testdb'
```
Comment (maxcao13) [inline backend/.env:1]: I think .env files should be in .gitignore?

File: backend/simple_db_test/test_model.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    const user = new userModel(userInfo);
+    try {
+        await user.save();
+        console.log(user);
+      } catch (error) {
+        console.log(error);
+      }
+}
+
+mongoose.connect('mongodb://localhost:27017/testdb',
```
Comment (maxcao13) [inline backend/simple_db_test/test_model.js:14]: This should probably use the process.env variable like thie server.js file

File: backend/.env (hunk 1 above)
Comment (dlalaj) [reply backend/.env:1]: Yes will add. Although I wanted to have everyone be able to pull the file at first since its being referenced in `server.js`

**Review by maxcao13 — APPROVED** [review]
LGTM!

**Pull request merged**